server/server/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
```

Comment:
You should name the directory something else than "server". This should be the name of your project.

Maybe "cantina" or "community_portal".

Reply:
Sorry @alexgarel I forgot to tell you that we can't rename the folder in django , if we have to rename than I have to initialize the django again